R/install.R

```diff
@@ -53,10 +55,16 @@ install <- function(collection = c('core', 'natverse'), pkgs=NULL,
       c("nat", "nat.nblast", "nat.templatebrains")
     else
       "natverse"
+    repos = paste0("natverse/", pkgs)
   } else {
     collection=NULL
+    #use the actual repo if it is suggested otherwise assume natverse..
+    if (grepl("/",pkgs)){
```

Comment:
Thanks a lot @SridharJagannathan! I think pkgs may have length >1 so I would suggest something like
```
pkgs=ifelse(grepl("/", pkgs), pkgs, paste0('natverse/', pkgs))
```
in place of this if else statement